src/session/components/message_entry.rs

```diff
@@ -0,0 +1,151 @@
+use glib::clone;
```

Comment:
I think the creation of this file has gone into the wrong commit.

Reply:
Why? The commit is the one about creating the MessageEntry widget.

Reply:
Tagged the wrong file. In fact, I mean `src/session/content/send_photo_dialog.rs`.

Reply:
No idea how did it end up there.